docs/source/getting-started/uninstalling.adoc

```diff
@@ -22,7 +22,7 @@ $ minishift delete
 +
 This command deletes everything in the
 `MINISHIFT_HOME/.minishift/machines/minishift` directory. Other cached data and
-the https://minishift.io/docs/managing-minishift.html#persistent-configuration[persistent configuration] are not removed.
+the link:../using/managing-minishift.html#persistent-configuration[persistent configuration] are not removed.
```

Comment:
this really should be an xref, but we take this discussion later once we settled the doc writing guidelines.